pkg/config/validation/validation.go

```diff
@@ -2359,9 +2359,9 @@ func analyzeUnreachableHTTPRules(routes []*networking.HTTPRoute,
 				duplicateMatches++
 				// no need to handle for totally duplicated match rules
 				continue
-			} else {
```

Comment:
[revie rule]superfluous-else: if block ends with a continue statement, so drop this else and outdent its block